ios/Podfile.lock

```diff
@@ -247,7 +247,7 @@ PODS:
     - React
   - RNCMaskedView (0.1.5):
     - React
-  - RNCPushNotificationIOS (1.4.0):
+  - RNCPushNotificationIOS (1.2.0):
```

Comment:
Why is this happening?

Reply:
did you pod install?

Reply:
@devinjameson  I think this might be because the global cocoa pods version on your system could be lower than the one bundled? Is the output from `gem which cocoapods` showing the `1.9.1` version or the `1.9.3` ?